ocs_ci/ocs/platform_nodes.py

```diff
@@ -1175,6 +1193,70 @@ def get_stack_name_of_node(self, node_name):
         stack_name = self.aws.get_stack_name_by_instance_id(instance_id)
         return stack_name
 
+    def wait_for_nodes_to_stop(self, nodes):
+        """
+        Wait for the nodes to reach status stopped
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+
+        Raises:
+            ResourceWrongStatusException: In case of the nodes didn't reach the expected status stopped.
+
+        """
+        instances = self.get_ec2_instances(nodes)
+        assert (
```

Comment:
It would be better to not have assert statements in a library with functions and throw an exception instead. Although, it is used in other functions there.

Reply:
Fixed